forcedphot.rst

```diff
@@ -0,0 +1,181 @@
+    +------+-------+--------------------+----------------+
+    | y    |    14 |              1.124 |         367.64 |
+    +------+-------+--------------------+----------------+
+
+Each is a per-object metric rather than a per-source metric, and therefore does not appear in the forced source table.
+This kind of analysis and subsequent plots should be written as a Pipeline Task since it is not feasible to run for more than a few thousand objects at a time within an analysis notebook.
+
+In Section :ref:`section-big-data-big-memory`, we show some aggregate results from inspecting fluxes in a handful of patches in the three tracts described above.
+In Section :ref:`section-light-curve-sample`, we describe a closer look at a very small subset of object light curves.
+Finally, we discuss the additional tooling and analysis work necessary to more fully vet forced photometry light curve data products.
+
+.. _section-big-data-big-memory:
+
+Big data, big memory challenges
+-------------------------------
+
+The forced source tables exist on a per-patch basis. While each tract has 49 patches of data, loading even the bare minimum of columns into memory (via pandas for analsis in a jupyter notebook, for instance) is prohibitive for more than a few patches at a time.
+As a result, the most data we aggregated at one time is from 5 patches for one of the shallower tracts.
+Distributions of the forced PSF flux for all sources in patches 0--4 of tract 3898 are shown in :numref:`fig-flux-hist-3828`.
+We examined a single patch's forced difference flux for all sources as well, shown in :numref:`fig-diff-flux-hist-3828-47`.
+Nothing unusual is apparent in these plots, but they also do not provide the kind of granular object-level light curved vetting desired.
+
+.. figure:: _static/flux-hist-3828.png
+    :name: fig-flux-hist-3828
+    :target: ../_images/flux-hist-3828.png
+    :scale: 40 %
+    :alt: Six panels with flux histograms by band.
+
+    Forced flux distributions by band for all sources in tract 3828, patches 0--4.
+
+.. figure:: _static/diff-flux-hist-3828-47.png
+    :name: fig-diff-flux-hist-3828-47
+    :target: ../_images/diff-flux-hist-3828-47.png
+    :scale: 40 %
+    :alt: Six panels with difference flux histograms by band.
+
+    Forced flux distributions by band **on the difference images** for all sources in tract 3828, patch 47. Because most detected sources are not also difference imaging sources, most have a flux at or near zero.
+
+The memory problem is particularly pronounced when creating and examining light curves.
+For example, it is straightforward to write down metrics one may use to characterize light curve variability, or to search by sky coordinate for a known variable object of interest.
+It is much more challenging to meaningfully examine light curves for a representative sample of objects.
+
+.. _section-light-curve-sample:
+
+A (very small) sampling of light curves
```

Comment:
I'm surprised that all the light curves are brightest in y band. Is that real?

Reply:
I hadn't noticed this! I have nothing meaningful to say about it, though 🤷‍♀️